element.go

```diff
@@ -346,18 +348,29 @@ func readRawItem(d *dicomio.Decoder) ([]byte, bool) {
 
 // PixelDataInfo is the Element.Value payload for PixelData element.
 type PixelDataInfo struct {
-	Offsets []uint32 // BasicOffsetTable
-	Frames  [][]byte // Parsed images
```

Comment:
previously wasn't really "frames" because native frames would be parsed into a single frame as raw bytes.